site/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="gemi-nikk">
+  <meta name="description" content="A personal website featuring a bio, favorite things, a haiku, and useful links about .">
+  <title>kishan's CSD-112 Project</title>
+  <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <link rel="shortcut icon" href="/favicon.ico" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+  <!-- Found me!  -->
+</head>
+<body>
+<div class="width-wrap">
+  <header id="t   op">
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="links.html">Links</a></li>
+        <li><a href="glossary.html">Glossary</a> </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="Thumbnails-and-Alternative-Sizes">
+      <a href="images/myself.jpg" target="_blank">
+        <picture>
+          <source media="(max-width: 500px)" srcset="images/thumbnail-150.jpg">
+          <source media="(max-width: 501px)" srcset="images/thumbnail-300.jpg">
+          <img src="images/thumbnail-300.jpg" alt="Thumbnail">
+        </picture>
+      </a>
+    </section>
```

Comment:
> 4. `Lab 3 > Thumbnails and Alternative Sizes > img uses sizes attribute`
> 5. `Lab 3 > Thumbnails and Alternative Sizes > img srcset is setup correctly`
> 6. `Lab 3 > Thumbnails and Alternative Sizes > displays the 150px thumbnail when the viewport is under 500px`
> 7. `Lab 3 > Thumbnails and Alternative Sizes > displays the 300px thumbnail when the viewport is over 500px`

This content was expected in the `bio` section, here are the expected cases.
```suggestion
    <section id="Thumbnails-and-Alternative-Sizes">
      <a href="images/myself.jpg" target="_blank">
        <picture>
          <img src="images/thumbnail-300.jpg" alt="Thumbnail" sizes="(max-width: 500px) 150px, 300px, srcset="images/thumbnail-150.jpg, images/thumbnail-300.jpg">
        </picture>
      </a>
    </section>
```

> 8. `Lab 3 > Thumbnails and Alternative Sizes > click on the displayed thumbnail opens the full image`
> 9. `Lab 3 > Thumbnails and Alternative Sizes > alt attribute is supplied to the images`

Overriding as this is correct but was undetected due to being within the wrong section.